samples/dect/dect_phy/dect_shell/src/dect/mac/dect_phy_mac_shell.c

```diff
@@ -505,11 +533,18 @@ static int dect_phy_mac_rach_tx_cmd(const struct shell *shell, size_t argc, char
 		desh_error("Arguments without '-' not supported: %s", argv[argc - 1]);
 		goto show_usage;
 	}
+	if (params.get_mdm_temp &&
+	    ((strlen(params.tx_data_str) + DECT_PHY_MAC_RACH_TX_DATA_JSON_OVERHEAD) >=
+	     (DECT_DATA_MAX_LEN - 1))) {
+		desh_error(
+			"Generated json data is too long to be included with modem temperature.");
```

Comment:
Maybe something like this. Or at least make JSON upper case.
```suggestion
			"The given data is too long to be encoded into JSON with modem temperature.");
```